normalization/src/main/java/zemberek/normalization/WordSegmenter.java

```diff
@@ -0,0 +1,88 @@
+package zemberek.normalization;
+
+import java.io.IOException;
+import java.util.*;
+import java.util.stream.Collectors;
+
+import zemberek.morphology.TurkishMorphology;
+import zemberek.morphology.analysis.WordAnalysis;
+
+public class WordSegmenter {
```

Comment:
Add small doc. To class or public methods. If you add some tests it is fine, I will check algortihm when I have time.